hbase-server/src/main/java/org/apache/hadoop/hbase/favored/FavoredNodeLoadBalancer.java

```diff
@@ -89,7 +89,7 @@ public synchronized void initialize() throws HBaseIOException {
   }
 
   @Override
-  public List<RegionPlan> balanceTable(TableName tableName,
+  public List<RegionPlan> balanceTablePlans(TableName tableName,
```

Comment:
Why these name changes?

Reply:
balanceTable() only generates plans without executing them. I add a method balanceRSGroup(), which generates plans and executes them.
For distinguishing these two meanings, I changed the method name.
If it is not necessary, I can remove these changes, thanks.